tests/cypress/integration/masthead_test.spec.js

```diff
@@ -0,0 +1,70 @@
+import ListingPage from "../support/pages/admin_console/ListingPage.js";
+import LoginPage from "../support/pages/LoginPage.js";
+import SidebarPage from "../support/pages/admin_console/SidebarPage.js";
+
+const loginPage = new LoginPage();
+
+const logOutTest = (userDropdown) => {
+  it("logs out", () => {
+    cy.get(userDropdown).click();
+    cy.get("#sign-out").click();
+    loginPage.isLogInPage();
+  });
+};
+
+const goToAcctMgtTest = (userDropdown) => {
+  it("opens manage account and returns to admin console", () => {
+    cy.get(userDropdown).click();
+    cy.get("#manage-account").click();
```

Comment:
It is better to use PageObjects to reuse code, this two lines can be replaced by
```suggestion
    headerPage.goToManageAccount();
```